app/src/test/java/net/squanchy/service/firestore/FirestoreFixtures.kt

```diff
@@ -0,0 +1,122 @@
+package net.squanchy.service.firestore
+
+import com.google.firebase.firestore.GeoPoint
+import net.squanchy.service.firestore.model.conferenceinfo.FirestoreConferenceInfo
+import net.squanchy.service.firestore.model.conferenceinfo.FirestoreVenue
+import net.squanchy.service.firestore.model.schedule.FirestoreDay
+import net.squanchy.service.firestore.model.schedule.FirestoreEvent
+import net.squanchy.service.firestore.model.schedule.FirestorePlace
+import net.squanchy.service.firestore.model.schedule.FirestoreSchedulePage
+import net.squanchy.service.firestore.model.schedule.FirestoreSpeaker
+import net.squanchy.service.firestore.model.schedule.FirestoreTrack
+import java.util.Date
+
+fun fixtureOfFirestoreTrack(
```

Comment:
I love these fixtures 💯 Could we name these `aFirebaseTrack`, `aFirestoreSpeaker` (and so on) for terseness?